### PR TITLE
Bitbucket new commit comment

### PR DESCRIPTION
**Pull request by jverce**
This PR is in support of issue #412
* Add event source logic for New Commit Comment

(Contains the changes from PR #481, so review that one first and then come back to this one)

### CODE REVIEW

File: components/bitbucket/bitbucket.app.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    _hookEndpointUrl(workspaceId, repositoryId, hookId) {
+      const baseUrl = this._hooksEndpointUrl(workspaceId, repositoryId);
+      // https://developer.atlassian.com/bitbucket/api/2/reference/meta/uri-uuid#uuid
+      return `${baseUrl}/{${hookId}}`;
+    },
+    async _propDefinitionsOptions(url, params, { page, prevContext }) {
+      let requestConfig = this._makeRequestConfig();  // Basic axios request config
+      if (page === 0) {
+        // First time the options are being retrieved.
+        // Include the parameters provided, which will be persisted
```
Comment (tjk) [inline components/bitbucket/bitbucket.app.js:89]: Do you just mean the nextPageUrl cursor will implicitly include the initial params?

Comment (tjk) [reply components/bitbucket/bitbucket.app.js:89]: (nitpicky, but either way this comment is a little unclear)

**Review by tjk — APPROVED** [review]
Looks good. Once again the `deleteHook` at least can probably be a little DRY'ed up a bit across components, but it's fine.

File: components/bitbucket/bitbucket.app.js (hunk 1 above)
Comment (jverce) [reply components/bitbucket/bitbucket.app.js:89]: Yes, that's exactly what I meant :slightly_smiling_face: 
I'll create a separate PR to rephrase it (similar logic with the same comment is in other components as well)

Comment (jverce) [reply components/bitbucket/bitbucket.app.js:89]: Added a task for this: https://github.com/PipedreamHQ/pipedream/issues/508

Comment (jverce) [reply components/bitbucket/bitbucket.app.js:89]: Created task for this: https://github.com/PipedreamHQ/pipedream/issues/509

**Pull request merged**